docs/docs-content/automation/palette-cli/commands/pde.md

```diff
@@ -45,16 +52,17 @@ subcommands.
+  | -------------- | ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+  | -              | `---app-profile-name`     | Install using a configuration file (optional). Use `-config-only` to generate a configuration file.                                                                                                                                                                                                                                                                         | string   |
+  | -              | `--app-profile-version`   | Generate configuration file only. This command will not proceed with installation.                                                                                                                                                                                                                                                                                          | boolean  |
+  | -              | `--cluster-group-name`    | Validate prerequisites for environment. Do not proceed with installation.                                                                                                                                                                                                                                                                                                   | boolean  |
+  | -              | `--cpu`                   | CPUs (optional). Defaults to 4 cores. (default 4)                                                                                                                                                                                                                                                                                                                           | int      |
+  | `-k`           | `--encryption-passphrase` | Encryption passphrase to secure sensitive data. The passphrase must be between 8 to 32 characters long and contain a capital letter, a lowercase letter, a digit, and a special character. Can be set through the environment variable `PALETTE_ENCRYPTION_PASSWORD`. Refer to the [Encryption](./../palette-cli.md#encryption) section for more information on encryption. | string   |
+  | -              | `--memory`                | Memory (GB) (optional). Defaults to 4GB. (default 4)                                                                                                                                                                                                                                                                                                                        | int      |
```

Comment:
```suggestion
  | -              | `--memory`                | Memory allocated to the App deployment (optional) in GB. Defaults to 4GB.                                                                                                                                                                                                                                                                                                                | int      |
```
Just a suggestion. I don't feel like we need to specifically say "default 4" since we already say that the value is an int. It feels like we are saying the same thing twice.